tests/run-fedora

```diff
@@ -1,6 +1,11 @@
 #!/bin/sh
 set -eux
 # install build dependencies
+source /etc/os-release
+if [ $VERSION_ID == 34 ]; then
+    dnf -y install dnf-plugins-core
+    dnf -y copr enable hadess/power-profiles-daemon # FIXME remove when using Fedora 35 or newer
+fi
```

Comment:
That's actually nice! (As an alternative, it would have been enough to run this for Rawhide only, and not install it on 34).